pkgs/development/tools/analysis/hotspot/default.nix

```diff
@@ -1,36 +1,42 @@
-{ lib,
-  mkDerivation,
-  cmake,
-  elfutils,
-  extra-cmake-modules,
-  fetchFromGitHub,
-  kconfigwidgets,
-  ki18n,
-  kio,
-  kitemmodels,
-  kitemviews,
-  kwindowsystem,
-  libelf,
-  qtbase,
-  threadweaver,
+{ lib
+, mkDerivation
+, cmake
+, elfutils
+, extra-cmake-modules
+, fetchFromGitHub
+, kconfigwidgets
+, ki18n
+, kio
+, kitemmodels
+, kitemviews
+, kwindowsystem
+, libelf
+, qtbase
+, threadweaver
+, qtx11extras
+, zstd
+, kddockwidgets
+, rustc-demangle
 }:
 
 mkDerivation rec {
   pname = "hotspot";
-  version = "1.3.0";
+  version = "unstable-2021-08-13";
```

Comment:
Do we want to update `hotspot` to an unstable version?

Usually nixpkgs packages release versions, unless the upstream project switches to unstable-only releases.

Reply:
I thought hotspot changed to unstable-only since I saw "continuous build" as the latest version. Is that not the case?

Reply:
I don't know, some projects CI is just set up to post its unstable releases onto the Github releases page even without setting the prerelease flag. Arch Linux apparently still packages 1.3.0. Might be worth asking upstream.

Reply:
OK, I won't update hotspot then.

Reply:
@1000teslas In that case I propose to conver this PR into a `draft PR`, so that we can reuse it and your changes when the next hotspot release comes out, and so that people who need the Rust support before that release have an easy way to get it.